bin/pycbc_live

```diff
@@ -0,0 +1,444 @@
+#!/usr/bin/env python]
+import argparse, numpy, numpy.random, pycbc, logging, cProfile, h5py, lal
+from pycbc import fft, version, waveform, types, filter as pfilter, scheme, psd as pypsd, noise, vetoes, strain, frame
+from pycbc.types import MultiDetOptionAction, zeros
+from pycbc.filter import LiveBatchMatchedFilter
+from time import time
+import os.path
+from mpi4py import MPI as mpi
+from pycbc.events import newsnr
+from pycbc.events.coinc import LiveCoincTimeslideBackgroundEstimator
+from pycbc.events.single import LiveSingleFarThreshold
+from pycbc.io.live import SingleCoincForGraceDB
+import pycbc.waveform.bank
+
+def makedir(path):
```

Comment:
I've swapped the order so this applies the check in the right way. I'm not sure I want to pull in a Pegasus.DAX3 dependency here, but that does highlight a compartmentalization we should do in pycbc.workflow to separate out the things that don't require DAX3 support so we can import that alone.